maps/chronosphere/locomotive.lua

```diff
@@ -481,12 +431,21 @@ function Public.set_player_spawn_and_refill_fish()
 	if not objective.locomotive_cargo[1] then return end
 	local cargo = objective.locomotive_cargo[1]
 	if not cargo.valid then return end
-	cargo.get_inventory(defines.inventory.cargo_wagon).insert({name = "raw-fish", count = math_random(1, 2)})
+	cargo.get_inventory(defines.inventory.cargo_wagon).insert({name = "raw-fish", count = 1})
 	local position = cargo.surface.find_non_colliding_position("stone-furnace", cargo.position, 16, 2)
 	if not position then return end
 	game.forces.player.set_spawn_position({x = position.x, y = position.y}, cargo.surface)
 end
 
+function Public.award_coins(_count)

```

Comment:
what if wagon is full? if coins awarded after jump, better to put them into outside spawned chests, as those are fresh and always empty.

Reply:
Considered it, but it's actually better this way because it stops one player quickly taking all of them (and creates a small dynamic around having to unload them fast enough)

I playtested and found that being full wasn't a problem, the wagon can hold >25k coins so if it gets full its the team's own fault

Reply:
IT IS STILL PROBLEM, because wagon can be full of OTHER items. Also having 100k coins being exported by inserter for next 50 minutes is just bad. One player taking the coins is no issue, that's how actually coins are usually used.

Reply:
It has way less coins than that. But I'm missing something: why would the first wagon be full of other items? Players can only use inserters to put them in, in which case they'd have no trouble taking them out again.

Reply:
Fish autogenerate in it...

Reply:
Yeah apart from that, because that won't make the wagon full?